src/Aquifer.Jobs/Aquifer.Jobs.csproj

```diff
@@ -32,6 +32,13 @@
             <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
             <CopyToPublishDirectory>Never</CopyToPublishDirectory>
         </None>
+        <None Update="appsettings.json">
+            <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+        </None>
+        <None Update="appsettings.Development.json">
```

Comment:
If you remove your local `appsettings.Development.json` (copy it to a different folder outside the solution) will the project still build?  This file won't be available for CI/CD and we don't want it checked into source control.

Reply:
We might need to use a post-build step instead:
```
xcopy /y $(ProjectDir)appsettings.Development.json $(ProjectDir)$(OutDir)
```

Reply:
It builds when I remove/relocate it.